Repositories/UserV1Service.cs

```diff
@@ -0,0 +1,51 @@
+using rexfinder_api.Models;
+using rexfinder_api.Migrations;
+
+namespace rexfinder_api.Repositories;
+
+public class UserV1Service : IUserV1Service
+{
+
+    private readonly MyPlacesDbContext _context;
+
+    public UserV1Service(MyPlacesDbContext context)
+    {
+        _context = context;
+    }
+
+
+    public IEnumerable<UserV1> GetAllUsers()
+    {
+        return _context.Users.ToList();
+    }
+
+    public UserV1 GetUserById(int userId)
+    {
+        return _context.Users.SingleOrDefault(c => c.UserId == userId);
+    }
+
+    public UserV1 UpdateUser(UserV1 newUser)
+    {
+        var originalUser = _context!.Users.Find(newUser.UserId);
+        if (originalUser != null)
+        {
+            //Not sure if this is where we update Username and Password.
```

Comment:
I think it would be safe to add username, but not password since we haven't implemented the password reset functionality with auth yet.

Reply:
Added username